### PR TITLE
Update announcements.json

### PR DESCRIPTION
**Pull request by TamGIBill**
Added Webinar Slides Are Available for the VBA Education March SCO/SAA Office Hours - COVID-19 Announcement

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by jhonny218 — APPROVED** [review]

File: src/applications/static-pages/school-resources/constants/announcements.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+  {
+    "name": "Webinar Slides Are Available for the VBA Education March SCO/SAA Office Hours - COVID-19",
+    "url": "https://www.benefits.va.gov/gibill/#20200330",
+    "date": "2020-03-31",
+    "displayStartDate": "2020-03-31",
+    "displayEndDate": "2020-04-31"
```
Comment (shawkey-daniel) [inline src/applications/static-pages/school-resources/constants/announcements.json]: There is no such thing as April 31st :)

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/static-pages/school-resources/constants/announcements.json (hunk 1 above)
Comment (jhonnyoddball) [reply src/applications/static-pages/school-resources/constants/announcements.json]: oh yeah, nice catch

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Comment by TamGIBill** [comment]
DOH!

Tammy S. Hurley
Education Service
Veterans Benefits Administration
202-461-9838

[bye3]

**Pull request closed without merging**